### PR TITLE
Grant 67 project estimate form page

### PR DESCRIPTION
**Pull request by PeterPavlatos**
Created the Project Estimate Form page
Altered checkbox component

### CODE REVIEW

File: client/pages/form-pages/funding-project-info.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                  { label: 'Yes', value: 'yes' },
+                  { label: 'No', value: 'no' },
+                ]}
+              ></Radio>
+            </div>
+            <Field
+              name='totalEstimatedCost'
+              type='text'
+              label='A. Total estimated cost of project (brought forward from earlier)'
+              maxLength={300}
```
Comment (karthikktamilmani-freshworks) [inline client/pages/form-pages/funding-project-info.tsx:44]: Here the default value should be the value that they've already filled. Maybe should add it in TODO I guess

File: client/pages/form-pages/funding-project-info.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  return (
+    <Formik initialValues={initialValues} onSubmit={() => {}}>
+      {({ values }: FormikProps<FundingProjectCostInfoInterface>) => (
+        <Form className='flex justify-center'>
+          <div className='w-2/4 p-4 gap-y-6 bg-white flex flex-col items-center'>
+            <div className='mb-4 flex items-center flex-col'>
+              <h1 className='text-xl font-medium text-bcBluePrimary'>
+                {FormStepTitles[FormSteps.FUNDING_INFO]}
+              </h1>
+              <h3>Please proivde detailed information for your application.</h3>
```
Comment (karthikktamilmani-freshworks) [inline client/pages/form-pages/funding-project-info.tsx:25]: Typo - provide

File: client/pages/form-pages/funding-project-info.tsx (hunk 1 above)
Comment (PeterPavlatos) [reply client/pages/form-pages/funding-project-info.tsx:44]: K sounds good.

File: client/pages/form-pages/funding-project-info.tsx (hunk 2 above)
Comment (PeterPavlatos) [reply client/pages/form-pages/funding-project-info.tsx]: Fixed typo.

**Pull request merged**